src/components/inputs/localized-money-input/README.md

```diff
@@ -171,3 +196,121 @@ LocalizedMoneyInput.hasEmptyValue([
+      centAmount: 1200
+    },
+    {
+      currencyCode: 'AMD',
+      centAmount: 3300,
+    }
+  ],
+};
+
+// A function to convert a document to form values.
+const docToFormValues = aDoc => ({
+  // The parseMoneyValue function will turn a MoneyValue into a
+  // value the LocalizedMoneyInput component can handle ({currency: amount})
+  prices: LocalizedMoneyInput.parseMoneyValues(aDoc.prices),
+});
+
+// a function to convert form values back to a document
+const formValuesToDoc = formValues => ({
+  // The convertToMoneyValue function will turn a LocalizedMoneyInput
+  // value into a value the API can handle
+  // It automatically converts to centPrecision or highPrecision
+  // depending on the number of supplied fraction digits and the
+  // used currency code.
+  // If you want to forbid highPrecision, then the form's validation
+  // needs to add an error when it sees a highPrecision price.
+  // See example below
+  prices: LocalizedMoneyInput.convertToMoneyValues(formValues.prices),
+});
+
+const validate = formValues => {
+  const errors = { prices: {} };
+  Object.keys(formValues.prices).forEach(currency => {
+    errors.prices[currency] = {};
+  })
+  const emptyCurrencies = LocalizedMoneyInput.getEmptyCurrencies(formValues.prices);
+  // ['ERU', 'USD']
+  // THis form dosen't accept highprecision prices
+  const highPrecisionCurrencies = LocalizedMoneyInput.getHighPrecisionCurrencies(formValues.prices);
+  // ['CAD', 'USD']
+  emptyCurrencies.forEach(emptyCurrency => {
+    errors.prices[emptyCurrency].missing = true;
+  });
+
+  highPrecisionCurrencies.forEach(highPrecisionCurrency => {
+    errors.prices[highPrecisionCurrency].highPrecision = true;
+  });
+
+  return omitEmpty(errors);
+};
+const initialValues = docToFormValues(doc);
+const renderErrors = (errors) => {
+  Object.keys(errors.prices).reduce(currency => {
+    const error = errors.prices[currency]?.missing && (
+                  <ErrorMessage>This field is required!</ErrorMessage>
+                ) ||
+                errors.prices[currency]?.highPrecision && (
+          <ErrorMessage>High precision prices not  supported here!</ErrorMessage>
+        ))
+    return {
+      [currency]: touched.prices[currency] &&  error
+    }
+  })
+}
+
+return (
+  <Formik
+    initialValues={initialValues}
+    validate={validate}
+    onSubmit={formValues => {
+      // doc will contain "prices" holding a MoneyValue,
+      // ready to be consumed by the API
+      const nextDoc = formValuesToDoc(formValues);
+      console.log(nextDoc);
+    }}
+    render={({
+      values,
+      errors,
+      touched,
+      setFieldValue,
+      setFieldTouched,
+      handleSubmit,
+      isSubmitting,
+    }) => (
+      <form onSubmit={handleSubmit}>
+        <LocalizedMoneyInput
+          value={values.prices}
+          onBlur={() => setFieldTouched('prices')}
+          isDisabled={isSubmitting}
+          onChange={value => setFieldValue('prices', value)}
```

Comment:
In the example, I'd rather do 

```
<LocalizedMoneyInput
  name="prices"
  onChange={formik.handleChange}
  onBlur={formik.handleBlur}
/>
```

instead of setting the name using `setFieldTouched` and `setFieldValue`.